togetherly/lib/models/person.dart

```diff
@@ -0,0 +1,22 @@
+
+
+enum FamilyRole {
+  parent,
+  child
+  // Add more statuses as needed
+}
+
+class Person {
```

Comment:
The Person model isn't something we need right away. However, we may want to have Parent and Child as separate subclasses of Person, rather than differentiate them based on a role enum. This is because Child will likely need to have additional fields for things like their point balance, and Parents can't be assigned chores. They could even be two entirely separate tables in the database.